types/react-loadable/test/index.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import Loadable, { LoadingComponentProps } from 'react-loadable';
+import * as Loadable from 'react-loadable';
```

Comment:
`import Loadable = require('react-loadable')`

The `React` import probably should be changed to this import style as well.

Reply:
Is this a stylistic preference or are there other technical reasons?

Reply:
`import * as Foo from 'foo'` is known to be incorrect when `foo` is a commonjs module; it just happens to work on both babel and typescript because they violate the spec, having been written before the spec was finalized.

`import Foo from 'foo'` will work at runtime with real modules, but in typescript it requires the flag to allow synthetic modules; since `.d.ts` files can't require flags to work, the typescript syntax for commonjs requires can be used (`import Foo = require('foo')` is the same as `const Foo = require('foo')` at runtime but only the version written with `import Foo =` gets the correct types)